examples/nanoChatGPT/config/train_rlhf.yaml

```diff
@@ -1,5 +1,5 @@
 # training
-episode_length: 25
+episode_length: 10
```

Comment:
we can reset this value to 25 if you think it is appropriate

Reply:
10 tokens is very short. In practice you would generate way more than 10 tokens (e.g. at most 10 words)